src/searchengine/nova/engines/btdigg.py

```diff
@@ -46,9 +46,6 @@ def search(self, what, cat='all'):
 
                     info_hash, name, files, size, dl, seen = line.strip().split('\t')[:6]
                     name = name.replace('|', '')
-                    # BTDigg returns unrelated results, we need to filter
-                    if not all(word in name.lower() for word in what_list):
```

Comment:
Long story short, this is where all the history has started. We move unconditional filtering from here into the qBt GUI and make it conditional. You can read explanations from @ngosang in the issue #3812.